app/assets/stylesheets/provider/admin/cms/_cms_intro.scss

```diff
@@ -80,22 +80,27 @@ $glow-color: $highlight-color;
 
 #cms-sidebar {
   li.glowing.active, li.glowing {
-    color: $glow-color !important;
+    background-color: $wildSandGray;
```

Comment:
Instead of using $wildSandGray directly add `$glow-background-color` to `colors.scss`